### PR TITLE
Update test runner

### PR DESCRIPTION
**Pull request by juannyG**
- Integrate tox
- Updates for Python 2.6 support

@marcosmoyano @gerardohuck  @eugene-og

### CODE REVIEW

File: tests.py [hunk 1]
```diff
@@ -12,28 +12,28 @@ def setUp(self):
         self.msg = "test"
 
     def test_run_check(self, mock_sys):
-        with self.assertRaises(NotImplementedError):
-            self.plugin_inst.run_check()
+        self.assertRaises(NotImplementedError,
+                          lambda: self.plugin_inst.run_check())
```
Comment (marcosmoyano) [inline tests.py]: why the lambda and not just self.plugin_inst.run_check (without the parenthesis)?

Comment (juannyG) [inline tests.py]: Because I followed the advice from stack-overflow late at night instead of RTFM :-)

Updating...stay tuned!

**Comment by juannyG** [comment]
updated per your comments @marcosmoyano

**Comment by marcosmoyano** [comment]
:+1:

**Comment by gerardohuck** [comment]
Too late, but this looks good nevertheless! :+1:

**Pull request merged**